packages/docs/src/routes/qwikcity/routing/index.mdx

```diff
@@ -50,8 +50,8 @@ src/
 Special named directories with square brackets, such as `[paramName]` and `[...catchAll]` can be used to match route segments which are dynamic:
 
 ```
-src/routes/blog/index.tsx → /blog/:slug (/blog/hello-world)
-src/routes/user/[username]/index.tsx → /user/:username/settings (/user/foo/)
```

Comment:
`/settings` part never existed, so I removed it. Also, example `(/user/foo/)` never had this route part.